app/Post.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Post extends Model
+{
+    // Table Name
+    protected $table = 'posts';
+    // Primary Key
+    public $primaryKey = 'id';
+    // Timestamps
```

Comment:
What additional information these comments give us?

Reply:
we can add these information to set our table name or to set our primary key  in our order